### PR TITLE
Measure whether we receive media and add the mute state as an exception

### PR DESCRIPTION
**Pull request by EnricoSchw**
We want to measure whether we receive media. This value serves as an indicator that the media pipeline is healthy. But in case of video is muted we don't get any bytes. Therefore we check the mute state of a track and treat it as if we received bytes. This pull request contains the calculation of muted track states. The determination remote track is mutes is made using the receiver track of a WbRTC connection.

## Checklist

-   [x] Tests written for new code (and old code if feasible)
-   [x] Linter and other CI checks pass
-   [x] Sign-off given on the changes (see [CONTRIBUTING.md](https://github.com/matrix-org/matrix-js-sdk/blob/develop/CONTRIBUTING.md))

---
This change is marked as an *internal change* (Task), so will not be included in the changelog.

### CODE REVIEW

File: src/webrtc/stats/media/mediaTrackStats.ts [hunk 1]
```diff
@@ -101,4 +104,28 @@ export class MediaTrackStats {
     public resetBitrate(): void {
         this.bitrate = { download: 0, upload: 0 };
     }
+
+    public set alive(isAlive: boolean) {
+        this.isAlive = isAlive;
+    }
+
+    public get alive(): boolean {
```
Comment (dbkr) [inline src/webrtc/stats/media/mediaTrackStats.ts:112]: The explanation of what 'alive' means could go here maybe?

**Review by dbkr — APPROVED** [review]
Otherwise lgtm

File: src/webrtc/stats/media/mediaTrackStats.ts (hunk 1 above)
Comment (EnricoSchw) [reply src/webrtc/stats/media/mediaTrackStats.ts:112]: Good point. I will add descriptions here.

**Comment by t3chguy** [comment]
Known Wysiwyg flake.

**Pull request merged**